modules/bitgo/test/browser/browser.spec.ts

```diff
@@ -33,6 +33,8 @@ describe('Coins', () => {
       XrpToken: 1,
       Rune: 1,
       Trune: 1,
+      Apt: 1,
```

Comment:
is this change needed?

Reply:
Yes, it is. Otherwise, one of the Browser Test case was failing previously.